include/mbedtls/pk.h

```diff
@@ -633,42 +579,42 @@ int mbedtls_pk_verify_restartable(mbedtls_pk_context *ctx,
                                   mbedtls_pk_restart_ctx *rs_ctx);
 
 /**
- * \brief           Verify signature, with explicit selection of the signature algorithm.
- *                  (Includes verification of the padding depending on type.)
+ * \brief           Verify signature, selecting a specific algorithm.
+ *                  (Includes verification of the padding if applicable.)
```

Comment:
“includes verification of the padding” is an obsolete concept from back when textbook RSA was considered a cryptographic algorithm rather than a low-level building block. Please remove this clause while you're at it.